staging/cse/windows/kubeletfunc.ps1

```diff
@@ -224,15 +224,15 @@ function New-InfraContainer {
         $ContainerRuntime = "docker"
     )
     cd $KubeDir
-    $windowsVersion = (Get-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").ReleaseId
+    $windowsVersion = Get-WindowsVersion -buildNumber (Get-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").CurrentBuild
 
     # Reference for these tags: curl -L https://mcr.microsoft.com/v2/k8s/core/pause/tags/list
     # Then docker run --rm mplatform/manifest-tool inspect mcr.microsoft.com/k8s/core/pause:<tag>
 
     $clusterConfig = ConvertFrom-Json ((Get-Content $global:KubeClusterConfigPath -ErrorAction Stop) | Out-String)
     $defaultPauseImage = $clusterConfig.Cri.Images.Pause
 
-    $pauseImageVersions = @("1809", "1903", "1909", "2004", "2009", "20h2")
+    $pauseImageVersions = @("1809", "1903", "1909", "2004", "2009", "20h2", "ltsc2022")

```

Comment:
It is fine to remove "1903", "1909", "2004", "2009", "20h2"

Reply:
I remove all and just check the windowsVersion is null or empty, so that it can save our maintainence. Is it OK?